app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -53,12 +55,14 @@ def course_option
       def courses
         Course
           .current_cycle
+          .open_on_apply
+          .exposed_in_find
```

Comment:
`.open_on_apply` implies `exposed_in_find`: https://github.com/DFE-Digital/apply-for-teacher-training/blob/a9384c00d2c9da838d838dc0c32ef12c15aa611d/app/models/course.rb#L13